gammapy/modeling/models/core.py

```diff
@@ -401,12 +399,12 @@ def covariance(self, covariance):
 
     @property
     def parameters(self):
-        """Parameters (`~gammapy.modeling.Parameters`)"""
+        """Parameters as a `~gammapy.modeling.Parameters` object."""
         return Parameters.from_stack([_.parameters for _ in self._models])
 
     @property
     def parameters_unique_names(self):
-        """List of unique parameter names as model_name.par_type.par_name"""
+        """List of unique parameter names as model_name.par_type.par_name."""
```

Comment:
This docstring was not really clear to me until I look at the methods. Maybe adding `output as model_name.par_type.par_name` would be more explanatory ? What do you think ?
```suggestion
        """List of unique parameter names output as a list of string formatted as model_name.par_type.par_name."""
```